README.md

```diff
@@ -31,6 +31,21 @@ services.AddEventuateTramSqlProducer(dbSchemaName, (provider, o) =>
 	});
 ```
 
+### Configuration Properties
+- Customize the maximum amount of time (ms) to poll for a new event (default 100):
```

Comment:
should this be to wait for a new event? As stated it sounds like it will poll at some unknown interval for 100ms and then stop polling.

Reply:
OK, I spent some time improving the documentation around the Kafka consumer configuration properties